.jenkins/pipelines/pr.Jenkinsfile

```diff
@@ -10,12 +10,30 @@ APPROVED_AUTHORS = [
     'RRathna',
     'paulcallen',
     'radhikaj',
-    'rs--',
     'salsal97',
-    'Sahakait',
     'vtikoo'
 ]
 
+// Do not trigger a full CI build if changes contain only these ignored files
+IGNORED_DIRS = [
+    '.jenkins/infrastructure',
+    '.jenkins/docker',
+    '.azure_pipelines',
+    '.github',
+    'doc'
```

Comment:
```suggestion
    'docs'
```

Reply:
It is just called `doc` in this repository, so no change is needed here